test/unit_test/fft_test_utils.hpp

```diff
@@ -425,7 +425,20 @@ void run_test(const test_params& params) {
   decltype(host_reference_output) host_output(desc.get_output_count(params.dir), padding_value);
   decltype(host_reference_output_imag) host_output_imag(
       Storage == complex_storage::SPLIT_COMPLEX ? desc.get_output_count(params.dir) : 0, padding_value);
-  double tolerance = 1e-3;
+  double tolerance = 5e-2;
+
+#ifdef PORTFFT_LOG_DUMPS
+  std::cout << "host_input: ";
+  for (std::size_t t = 0; t < host_input.size(); ++t) {
+    std::cout << host_input[t] << ", ";
+  }
+  std::cout << std::endl;
+  std::cout << "host_input_imag: ";
+  for (std::size_t t = 0; t < host_input_imag.size(); ++t) {
+    std::cout << host_input_imag[t] << ", ";
+  }
+  std::cout << std::endl;
```

Comment:
we could use one function for all prints from the host, here as well as in global_dispatcher

Reply:
done